src/Controller/AuthController.php

```diff
@@ -244,18 +244,14 @@ public function login(Request $request) {
       $lockExtraSettings = "{}";
     }
 
-    $returnTo = NULL;
-    if ($request->request->has('returnTo')) {
-      $returnTo = $request->request->get('returnTo');
-    }
-    elseif ($request->query->has('returnTo')) {
-      $returnTo = $request->query->get('returnTo');
+    $returnTo = $request->request->get('returnTo', NULL);
```

Comment:
This would be the same. (remove the if below) Does it look better to you or do you rather use the if?

```suggestion
    $returnTo = $request->request->get('returnTo', $request->query->get('returnTo', NULL));
```